src/components/pages/Account.tsx

```diff
@@ -20,32 +28,69 @@ type DispatchProps = {
 
 type StateProps = {
   loggedInMember: Member;
+  trustedForRecovery: Member;
+  votingFor: Member;
+};
+
+type Props = DispatchProps & OwnProps & StateProps;
+
+type State = {
+  timeRemaining: string;
 };
 
-type Props = DispatchProps & OwnProps;
+const Break: React.StatelessComponent<{}> = () => <View style={styles.break} />;
 
 // TODO show when account will go inactive, total Raha outstanding,
 // leaderboard, who you are voting for, etc.
 // Eventually it could be AccountTab instead of ProfileTab.
-const AccountView: React.StatelessComponent<Props> = props => (
-  <View style={styles.container}>
-    <Button title="Sign Out" onPress={props.signOut} />
-  </View>
-);
+class AccountView extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      timeRemaining: this.getTimeRemaining()
+    };
+  }
+
+  private getTimeRemaining() {
+    // TODO make a RED countdown with finer granularity than 1 day
+    // if you're within a month of being marked inactive, call
+    // updateTimeRemaining on setInterval.
+    const sinceLastMintMilli =
+      Date.now() - this.props.loggedInMember.lastMinted.getTime();
+    return `${400 - Math.round(sinceLastMintMilli / MS_PER_DAY)} days`;
+  }
+
+  render() {
+    const { signOut, trustedForRecovery, votingFor } = this.props;
+    const { timeRemaining } = this.state;
+    return (
+      <ScrollView>
+        <Text>
+          After {<Text style={styles.timeRemaining}>{timeRemaining}</Text>}{" "}
+          without activity your account balance will be donated.
+        </Text>
+        <Break />
+        <Text>Your Raha Parliament vote goes to:</Text>
+        <MemberThumbnail member={votingFor} />
+        <Break />
+        <Text>Trusted for account recovery:</Text>
+        <MemberThumbnail member={trustedForRecovery} />
+        <Break />
+        <Button title="Sign Out" onPress={signOut} />
+      </ScrollView>
+    );
+  }
+}
 
 const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    justifyContent: "center",
-    alignItems: "center",
-    backgroundColor: colors.lightBackground
-  },
-  bar: {
-    width: "100%",
-    height: 30
+  break: {
+    backgroundColor: colors.border2,
+    height: 5,
+    marginTop: 2,
+    marginBottom: 2
   },
-  barText: {
-    fontSize: 20
+  timeRemaining: {
+    fontWeight: "800"
```

Comment:
This doesn't apply to both Android/iOS equally I think. In fonts.ts we don't actually support 800 for any of our fonts. Maybe use 'normal', 'bold' instead?